cloudify_cli/cli/helptexts.py

```diff
@@ -559,7 +559,8 @@
 SEARCH_NAME = 'Search deployments by their display name. The returned list ' \
               'will include only deployments that contain the given search ' \
               'pattern'
-
+DEPENDENCIES_OF = 'List only deployments on which a given deployment ID ' \
+                  'depends.'
```

Comment:
i'd have said it "**the** given" or even "on which this deployment depends", or just "dependencies of this/the given deployment"